### PR TITLE
require extensions in imports

### PR DESCRIPTION
**Pull request by snapwich**
## Type of change
- [x] Refactoring (no functional changes, no api changes)

## Description of change
Explicitly require extensions on imports and validate with eslint plugin.

## Other information
This should fix any issues with dependent projects including Prebid.js that don't have the default extensions as part of the module resolver as described in #4856

### CODE REVIEW

File: modules/rubiconBidAdapter.js [hunk 1]
```diff
@@ -638,7 +638,7 @@ export const spec = {
               bidObject.adserverTargeting = extPrebidTargeting;
             }
 
-            // try to get cache values from 'response.ext.prebid.cache'
+            // try to get cache values from 'response.ext.prebid.cache.js'
```
Comment (pm-harshad-mane) [inline modules/rubiconBidAdapter.js]: is this change necessary?

File: test/spec/videoCache_spec.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
-import 'mocha/mocha';
```
Comment (robertrmartinez) [inline test/spec/videoCache_spec.js]: Guessing this is an unused import

**Comment by pm-harshad-mane** [comment]
Looks good to me, but please check the failing CI Job.

File: test/spec/videoCache_spec.js (hunk 2 above)
Comment (snapwich) [reply test/spec/videoCache_spec.js]: I sure hope so, I can't imagine what it would be doing where it'd need to be included only for its side-effects... either way I removed it and the tests still passed so... good?

Comment (robertrmartinez) [reply test/spec/videoCache_spec.js]: 👍

File: modules/rubiconBidAdapter.js (hunk 1 above)
Comment (snapwich) [reply modules/rubiconBidAdapter.js]: nope, that was an accidental catch by the regex replace
